### DISCUSSION TITLE
单击进度条上的章节时切换播放状态

### DISCUSSION THREAD

**Discussion by tgjqr** [hooke007/mpv_PlayKit#483] [mpv 通用]
mpv.conf文件中包含`MBTN_LEFT   cycle pause`，在点击章节时会切换播放状态（播放/暂停）。同样的条件下[mpv-lazy-2024V0](https://github.com/hooke007/MPV_lazy/releases/tag/20240102)版本中无此现象。

![无标题](https://github.com/hooke007/MPV_lazy/assets/101369945/5f5aaa6f-51e2-443d-a344-fc50421580ce)

**Comment by hooke007** [comment]
To trace https://github.com/tomasklaen/uosc/discussions/931
